ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -8,6 +8,33 @@ QtObject {
 
     readonly property var userInputLocale: Qt.locale("en_US")
 
+    function integralPartLength(num) {
```

Comment:
Wondering if this simplified version will work the same or you see some drawbacks? Do you think for big numbers will int
```js
return Math.floor(Math.abs(num)).toString().length
```
I did some quick tests in the browser js console and it worked for some 20+ floating numbers

Reply:
This doesn't seem to work fine when JS uses scientific notation (`num >= 10**21`)

![image](https://user-images.githubusercontent.com/11161531/230922207-c0ac5907-4976-4827-9953-9259c98b611b.png)